radis/lbl/broadening.py

```diff
@@ -888,11 +893,17 @@ def _calc_broadening_HWHM(self):
         # diluent and their broadening coeff dictionary
         diluent_broadening_coeff = {}
         for key in diluent:
-            if key != "air":
+            try:
```

Comment:
this try/except is also introducing many room for failures. Do we really need it ?

Reply:
I think this is required as it just gives a warning in broadening coefficients are not present.

Reply:
ok !